sympy/polys/matrices/domainmatrix.py

```diff
@@ -119,6 +122,19 @@ def __rmul__(A, B):
         else:
             return NotImplemented
 
+    def __smul__(self, other):
```

Comment:
What is `__smul__`?

I don't think that is a special method name.

Reply:
It represents scalar multiplication.
Please suggest some meaningful name.
Thanks.